certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, acme_server, nodes, http_proxy=True, stdout=False, dns_server
         self._acme_type = 'pebble' if acme_server == 'pebble' else 'boulder'
         self._proxy = http_proxy
         self._workspace = tempfile.mkdtemp()
-        self._processes = []
+        self._processes = []  # type: List
```

Comment:
Is this typing really needed?

Reply:
I got a warning about it, but it seems the adjustment in `linter_plugin.py` made it obsolete. It's removed.